src/index.coffee

```diff
@@ -0,0 +1,90 @@
+EvalSha = require 'redis-evalsha'
+_ = require 'underscore'
+async = require 'async'
+fs = require 'fs'
+redis = require 'redis'
+
+module.exports = class RateLimit
+
+  constructor: (@redisClient, rules, @prefix = 'ratelimit') ->
+    @checkFn = 'check_rate_limit'
+    @checkIncrFn = 'check_incr_rate_limit'
+
+    @eval = new EvalSha @redisClient
+    @eval.add @checkFn, @checkLimitScript()
+    @eval.add @checkIncrFn, @checkLimitIncrScript()
+
+    @rules = @convertRules rules
+
+  readLua: (filename) ->
+    fs.readFileSync("#{__dirname}/../lua/#{filename}.lua").toString()
```

Comment:
If you invoke `readFileSync` with 'utf8' as the second argument it returns a string instead of a buffer, then you don't need to call `.toString`